lib/utils/docker.coffee

```diff
@@ -48,6 +48,12 @@ exports.appendOptions = (opts) ->
 			alias: 't'
 		},
 		{
+			signature: 'arg'
+			parameter: 'arg'
+			description: 'Set a build-time variable (eg. "-e \'ARG=value\'"). Can be specified multiple times.'
+			alias: 'e'
```

Comment:
We've been talking about making big breaking changes to `local push` anyway though, so we could make this inconsistent in the short-term and then break `local push` to match shortly.

We don't necessarily need to worry about consistency there, if we think we are changing that anyway, as long as we think this is the right thing to do. If we don't of course, that's a different question :smiley:.

If we can't do build-arg, `A` sounds like an ok compromise to me though.